### ISSUE TITLE
Enforcing RequireTableSchema for SELECT, INSERT, UPDATE, DELETE?

### ISSUE THREAD

**Issue by hfhchan-plb** [sbdchd/squawk#1278]
Is it possible to extend RequireTableSchema to enforce it for SELECT, INSERT, UPDATE, DELETE? Currently it seems enforced for CREATE TABLE, CREATE TABLE AS, ALTER TABLE, DROP TABLE only

**Comment by sbdchd** [comment]
Yeah I think this is doable. We can expand the existing rule

**Comment by sbdchd** [comment]
So specifically, you're saying:

```sql
select a, now() from t;
```

should error

but these would pass:

```sql
select t.a, now() from t;
```
